third_party/move/move-compiler/src/inlining/translate.rs

```diff
@@ -101,7 +102,7 @@ impl<'l> Inliner<'l> {
                 if !fdef.inline {
                     let mut visitor = OuterVisitor { inliner };
                     Dispatcher::new(&mut visitor).function(fdef);
-                    //Self::eprint_fdef(&format!("fun {} ", _name), fdef);
+                    //DEBUG Self::eprint_fdef(&format!("fun {} ", _name), fdef);
```

Comment:
Do we need this?

Reply:
Not the new comment.  I'll leave the old one, though.